File: DentiSmart.App/app/src/main/java/com/dreamteam/dentismart/models/Diente.kt
package com.dreamteam.dentismart.models

import java.io.Serializable

data class Diente(
    val centro: Int,
    val derecho: Int,
    val estado: Int,
    val izquierdo: Int,
    val norte: Int,
    val sur: Int
) : Serializable

File: DentiSmart.API/DentiSmart.Domain/Models/Diente.cs
using MongoDB.Bson.Serialization.Attributes;
using System;
using System.Collections.Generic;
using System.Text;

namespace DentiSmart.Domain.Models
{
    [BsonIgnoreExtraElements]
    public class Diente
    {
        public int Izquierdo { get; set; }
        public int Derecho { get; set; }
        public int Centro { get; set; }
        public int Norte { get; set; }
        public int Sur { get; set; }
        public int Estado { get; set; }
    }
}
